src/nodedef.cpp

```diff
@@ -1203,6 +1231,54 @@ void boxVectorUnion(const std::vector<aabb3f> &boxes, aabb3f *box_union)
 	}
 }
 
+/*!
+ * Helper function for fixed/leveled nodeboxes in getBoxUnionReturns.
+ * Returns the smallest box that contains all boxes
+ * in the vector. Box_union is expanded.
+ * @param[in]      features   used to decide whether the nodebox
+ *                            can be rotated
+ * @param[in, out] box_union  the union of the arguments
+ * @param          to_add     nodebox to add to union
+ * @param          nbt        node box type
+ */
+void rawUnionFixed(const ContentFeatures &features,
+		aabb3f *box_union,
+		const std::vector<aabb3f> &to_add, enum NodeBoxType nbt)
+{
+	// Raw union (fixed)
+	aabb3f half_processed(0, 0, 0, 0, 0, 0);
```

Comment:
This seems slightly suspicious to me: What if you have a min point larger than 0, or a max point less than 0? I would initialize this box to `{INFINITY, INFINITY, INFINITY, -INFINITY, -INFINITY, -INFINITY}` instead.